backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardQueryService.kt

```diff
@@ -5,13 +5,15 @@ import app.ehrenamtskarte.backend.exception.service.ProjectNotFoundException
 import app.ehrenamtskarte.backend.verification.database.CodeType
 import app.ehrenamtskarte.backend.verification.service.CardVerifier
 import app.ehrenamtskarte.backend.verification.webservice.schema.types.CardVerificationModel
+import app.ehrenamtskarte.backend.verification.webservice.schema.types.CardVerificationResultModel
 import com.expediagroup.graphql.generator.annotations.GraphQLDescription
 import graphql.schema.DataFetchingEnvironment
 import java.util.Base64
 
 @Suppress("unused")
 class CardQueryService {
-    @GraphQLDescription("Returns whether there is a card in the given project with that hash registered for that this TOTP is currently valid")
+    @Deprecated("Deprecated in favor of CardVerificationResultModel that return a current timestamp", ReplaceWith("verifyCardInProjectV2"))
```

Comment:
```suggestion
    @Deprecated("Deprecated since May 2023 in favor of CardVerificationResultModel that return a current timestamp", ReplaceWith("verifyCardInProjectV2"))
```